src/Controller/AuthenticationStatusController.php

```diff
@@ -111,28 +113,6 @@ private function refreshAuthenticationPage(): JsonResponse
         return $this->generateAuthenticationStatusResponse('needs-refresh');
     }
 
-    /**
-     * Check if the authentication challenge is expired.
-     *
-     * If the challenge is expired, the page should be refreshed so a new
-     * challenge and QR code is generated.
-     *
-     * @return bool
-     */
-    private function authenticationChallengeIsExpired(): bool
-    {
-        // The use of authenticationUrl() here is a hack, because it depends on an implementation detail
-        // of this function.
-        // Effectively this does a $this->_stateStorage->getValue(self::PREFIX_CHALLENGE . $sessionKey);
-        // To check that the session key still exists in the Tiqr_Service's state storage
-        try {
-            $this->tiqrService->authenticationUrl();
```

Comment:
This is the old logic, the story mentions. I opted to stop applying this method of testing for timeouts. And simply apply the same timeout check I added for the enrollment (registration) logic.